packages/wheels_driver/include/hardware_test_wheels/hardware_test_motor.py

```diff
@@ -0,0 +1,79 @@
+import rospy
+
+from enum import Enum, auto
+
+from dt_duckiebot_hardware_tests import HardwareTest, HardwareTestJsonParamType
+from wheels_driver.dagu_wheels_driver import DaguWheelsDriver
+
+
+class HardwareTestMotorSide(Enum):
+    LEFT = auto()
+    RIGHT = auto()
+
+
+class HardwareTestMotor(HardwareTest):
+    def __init__(
+        self,
+        wheel_side: "HardwareTestMotorSide",
+        motors_driver: DaguWheelsDriver,
+        vel: float = 0.5,
+        duration: int = 3,
+    ) -> None:
+        self._info_str = "left" if wheel_side == HardwareTestMotorSide.LEFT else "right"
+        super().__init__(service_identifier=f"tests/{self._info_str}")
+
+        # attr
+        self._driver = motors_driver
+        self._side = wheel_side
+
+        # test settings
+        self.vel = vel
+        self.duration = duration
+
+    def test_id(self) -> str:
+        return f"Motor ({self._info_str})"
+
+    def test_description_preparation(self) -> str:
```

Comment:
Place your Duckiebot upside down on a flat surface with its wheels in the air.  When you run the test, the wheels will start to spin.